### PR TITLE
[CCMV] Update CCM _index.md page to include billing conductor

### PR DESCRIPTION
**Pull request by crousseaux**
### What does this PR do?
Includes Billing Conductor section for AWS tab in CCM documentation
Preview: https://docs-staging.datadoghq.com/constance/ccm-docs-update/cloud_cost_management/

### Motivation
We recently added support for AWS Billing Conductor

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/cloud_cost_management/_index.md [hunk 1]
```diff
@@ -179,6 +179,13 @@ The following out-of-the-box tags are also available for filtering and grouping
 [4]: https://docs.aws.amazon.com/cur/latest/userguide/view-cur.html
 [5]: https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-console.html
 [6]: https://docs.aws.amazon.com/cur/latest/userguide/data-dictionary.html
+
+### Billing conductor
+Billing conductor enables you to simplify the bill by customizing the billing rates, distribute credits and fees, and shared overhead costs at your discretion. You will also be able to select which accounts to include in the CUR.
```
Comment (buraizu) [inline content/en/cloud_cost_management/_index.md:184]:
```suggestion
Billing conductor enables you to simplify your bill by customizing the billing rates, distributing credits and fees, and sharing overhead costs at your discretion. You can also select which accounts to include in the CUR.
```

File: content/en/cloud_cost_management/_index.md [hunk 2]
```diff
@@ -179,6 +179,13 @@ The following out-of-the-box tags are also available for filtering and grouping
...
+
+To create a billing conductor CUR, please follow [these instructions](https://docs.aws.amazon.com/cur/latest/userguide/cur-data-view.html) from AWS. Make sure the CUR meets [Datadog's requirements](https://docs.datadoghq.com/cloud_cost_management/?tab=aws#prerequisite-generate-a-cost-and-usage-report).
+Once the billing conductor CUR is created, please follow the instructions above to set it up in Datadog.
+
```
Comment (buraizu) [inline content/en/cloud_cost_management/_index.md:188]:
```suggestion
To create a billing conductor CUR, follow the [AWS Cost and Usage Reports user guide][1]. Ensure the CUR meets [Datadog's requirements][2].
Once the billing conductor CUR is created, follow the Cloud Cost Management instructions above to set it up in Datadog.

[1]: https://docs.aws.amazon.com/cur/latest/userguide/cur-data-view.html
[2]: https://docs.datadoghq.com/cloud_cost_management/?tab=aws#prerequisite-generate-a-cost-and-usage-report
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, left some minor feedback per our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md).

Separately, regarding this point:
`follow the Cloud Cost Management instructions above to set it up in Datadog`

Is there a particular step that a user should start at once the CUR has been created? For example, should they next [Configure Access](https://docs.datadoghq.com/cloud_cost_management/?tab=aws#configure-access-to-the-cost-and-usage-report)?

**Comment by crousseaux** [comment]
Once the CUR has been created, they should go to [Configure AWS integration](https://docs.datadoghq.com/cloud_cost_management/?tab=aws#configure-the-aws-integration)

Should I add the link?

File: content/en/cloud_cost_management/_index.md [hunk 3]
```diff
@@ -179,6 +179,15 @@ The following out-of-the-box tags are also available for filtering and grouping
 [4]: https://docs.aws.amazon.com/cur/latest/userguide/view-cur.html
 [5]: https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-console.html
 [6]: https://docs.aws.amazon.com/cur/latest/userguide/data-dictionary.html
+
+### Billing conductor
```
Comment (buraizu) [inline content/en/cloud_cost_management/_index.md:183]:
```suggestion
## Billing conductor
```

**Comment by buraizu** [comment]
@crousseaux thanks for confirming that detail. In that case, I don't think anything needs to be updated, as that is the next step after the already stated prerequisite.
With that said, I realized that in this case `Billing conductor` is being included as a level three heading (`###`) along with the setup instructions, when it should possibly be its own section (`##`, as it's not technically related to setup). Do you think that might make more sense for the layout of the page?

**Pull request merged**